$NetBSD: patch-dist_configure.in,v 1.2 2013/09/20 14:50:49 ryoon Exp $

First chunk: automake-1.13 compat.
Second: Fix build on SunOS with native cures, detect isblank in configure
	script.
Rest: ?

--- dist/configure.in.orig	2007-11-18 16:41:42.000000000 +0000
+++ dist/configure.in
@@ -4,7 +4,7 @@ dnl Process this file with autoconf to p
 AC_INIT(../common/main.c)
 AC_CONFIG_AUX_DIR(.)
 AM_INIT_AUTOMAKE(vi, 1.81.6)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 dnl Configure setup.
 AC_PROG_INSTALL()
@@ -601,6 +601,7 @@ AC_CHECK_FUNCS(select)
 AC_CHECK_FUNCS(setenv, [need_env=no], [need_env=yes])
 AC_CHECK_FUNCS(strsep, [need_strsep=no], [need_strsep=yes])
 AC_CHECK_FUNCS(unsetenv,, [need_env=yes])
+AC_CHECK_FUNCS(isblank)
 
 AC_FUNC_MMAP
 AC_FUNC_VFORK
@@ -787,6 +788,8 @@ if test "$vi_cv_gettimeofday" = broken;
 fi
 AC_MSG_RESULT($vi_cv_gettimeofday)
 
+AC_CHECK_HEADERS([sys/stropts.h stropts.h])
+
 dnl Check for which version of openpty to use, System V or Berkeley.
 AC_MSG_CHECKING(for System V pty calls)
 AC_CACHE_VAL(vi_cv_sys5_pty, [dnl
